index.html

```diff
@@ -105,27 +105,34 @@ <h1 class="brand-heading">Brain&nbsp;Imaging Data&nbsp;Structure</h1>
         <p>BIDS is heavily inspired by the format used internally by <a href="http://openfmri.org">OpenfMRI.org</a>. While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt. The specification is intentionally based simple file formats and folder structures to reflect current lab practices and make it accessible to wide range of scientists coming from different backgrounds.</p>
         <p>Software currently supporting BIDS:</p>
         <ul>
-          <li><a href="http://preprocessed-connectomes-project.org/quality-assessment-protocol/">QAP</a> (quality assesment)</li>
-          <li><a href="http://mriqc.readthedocs.org/">MRIQC</a> (quality assesment)</li>
-          <li><a href="https://github.com/poldracklab/preprocessing-workflow">FMRIPREP</a> (preprocessing workflow)</li>
-          <li><a href="https://github.com/rhodricusack/automaticanalysis">Automatic Analysis</a> (fMRI processing toolbox)</li>
-          <li><a href="https://github.com/INCF/openfmri2bids">OpenfMRI2BIDS</a> (converter)</li>
+          <li><a href="http://bids-apps.neuroimaging.io">BIDS Apps</a> (a growing set of portable containerized data processing pipelines that understand BIDS datasets)</li>
+          <li>Converters</li><ul>
+          <li><a href="https://github.com/nih-fmrif/BIDS-tools">AFNI BIDS-tools</a> (set of tools for converting to and analyzing BIDS datasets in AFNI)</li>
           <li><a href="https://github.com/INCF/BIDS2ISATab">BIDS2ISATab</a> (converter)</li>
```

Comment:
Adding sub lists is a great idea. I think we can drop the now redundant descriptions in brackets such as "(converter)".